### ISSUE TITLE
Feature: display some information after bootup about what to do next!

### ISSUE THREAD

**Issue by supersexy** [librenms/packer-builds#16]
Well, I just want to quickly check LibreNMS, found these Virtualbox images, booted them up and asked myself **"Now WHAT???"**. This could be avoided by showing _one_ sentence after bootup that shows e.g. the ip address and user / password for the web ui login. Or at least show a URL to the docs. Thanks!

**Comment by evo2k6** [comment]
Agreed with the original post.  I was able to successfully login however is there a user guide for the VM?

I'm basically at the prompt and have no direction to the next steps.

Any tips or links to get started would be greatly appreciated and would need very little "hand-holding" as I usually like to do things for myself.

Thanks!
E

**Comment by evo2k6** [comment]
I guess I should've looked with my eyes and NOT with my mouth.  Anyhow, thanks to the answer by the "TheGreatDoc", I got my answer

https://community.librenms.org/t/i-do-not-understand-how-to-use-the-vm/8337

**Comment by murrant** [comment]
It would be great to have that, pull requests welcome.
